### PR TITLE
Update Arabic translation

### PR DESCRIPTION
**Pull request by burhandodhy**
**Description**
This PR updates the Arabic translation.

**Jira link**
https://edlyio.atlassian.net/browse/EDLY-1364

### CODE REVIEW

File: conf/locale/ar/LC_MESSAGES/djangojs.po [hunk 1]
```diff
@@ -250,7 +273,7 @@ msgstr "إلغاء"
 #: cms/static/js/features/import/factories/import.js
 #: common/static/bundles/Import.js
 msgid "There was an error during the upload process."
-msgstr "حدث خطأ أثناء عملية التحميل."
+msgstr "{numErrors} تعذر إضافة الطلّاب إلى هذا الفوج:"
```
Comment (rehan99000) [inline conf/locale/ar/LC_MESSAGES/djangojs.po:276]: No placeholder in the source string

File: conf/locale/ar/LC_MESSAGES/djangojs.po [hunk 2]
```diff
@@ -260,7 +283,7 @@ msgstr "حدث خطأ أثناء تحديث هدفك."
 #: cms/static/js/features/import/factories/import.js
 #: common/static/bundles/Import.js
 msgid "There was an error while verifying the file you submitted."
-msgstr "تعذر إضافة الطلّاب إلى هذا الفوج:"
+msgstr "{numErrors} تعذر إضافة الطلّاب إلى هذا الفوج:"
```
Comment (rehan99000) [inline conf/locale/ar/LC_MESSAGES/djangojs.po:286]: same here

File: conf/locale/ar/LC_MESSAGES/djangojs.po [hunk 3]
```diff
@@ -270,28 +293,32 @@ msgstr "اختر الملف"
 #: cms/static/js/features/import/factories/import.js
 #: common/static/bundles/Import.js
 msgid "There was an error while importing the new library to our database."
-msgstr "حدث خطأ أثناء استيراد المكتبة الجديدة إلى قاعدة بياناتنا."
+msgstr "{numErrors} تعذر إضافة الطلّاب إلى هذا الفوج:"
```
Comment (rehan99000) [inline conf/locale/ar/LC_MESSAGES/djangojs.po:296]: same here

File: conf/locale/ar/LC_MESSAGES/djangojs.po [hunk 4]
```diff
@@ -270,28 +293,32 @@ msgstr "اختر الملف"
...
 
 #: cms/static/js/features/import/factories/import.js
 #: common/static/bundles/Import.js
 msgid "There was an error while importing the new course to our database."
-msgstr "حدث خطأ أثناء استيراد الدورة التدريبية الجديدة إلى قاعدة بياناتنا."
+msgstr "{numErrors} تعذر إضافة الطلّاب إلى هذا الفوج:"
```
Comment (rehan99000) [inline conf/locale/ar/LC_MESSAGES/djangojs.po:301]: and here

File: conf/locale/ar/LC_MESSAGES/djangojs.po [hunk 5]
```diff
@@ -270,28 +293,32 @@ msgstr "اختر الملف"
...
 
 #: cms/static/js/features/import/views/import.js
 #: common/static/bundles/Import.js
 msgid "There was an error with the upload"
-msgstr "حدث خطأ في التحميل"
+msgstr "{numErrors} تعذر إضافة الطلّاب إلى هذا الفوج:"
```
Comment (rehan99000) [inline conf/locale/ar/LC_MESSAGES/djangojs.po:306]: here

File: conf/locale/ar/LC_MESSAGES/djangojs.po [hunk 6]
```diff
@@ -592,7 +629,7 @@ msgstr "جاري الحذف"
 
 #: cms/static/js/views/utils/xblock_utils.js common/static/bundles/commons.js
 msgid "Delete this {xblock_type}?"
-msgstr "هل تريد حذف {xblock_type} هذا؟"
+msgstr "أتودّ حذف هذا الفريق؟"
```
Comment (rehan99000) [inline conf/locale/ar/LC_MESSAGES/djangojs.po:632]: missing placeholder in the translation

File: conf/locale/ar/LC_MESSAGES/djangojs.po [hunk 7]
```diff
@@ -3206,7 +3230,7 @@ msgstr "تسجيل الدخول"
 
 #: common/static/bundles/SingleSupportForm.js
 msgid "Create an {platform} account"
-msgstr "إنشاء حساب {platform}"
+msgstr "إنشاء حساب"
```
Comment (rehan99000) [inline conf/locale/ar/LC_MESSAGES/djangojs.po:3233]: here

File: conf/locale/ar/LC_MESSAGES/djangojs.po [hunk 8]
```diff
@@ -9150,7 +9175,7 @@ msgstr "تغيير كلمة المرور"
 
 #: lms/templates/student_account/register.underscore
 msgid "Already have an account?"
-msgstr "هل لديك حساب مسبقاً؟"
+msgstr "هل لديك حساب {platformName} مسبقاً؟"
```
Comment (rehan99000) [inline conf/locale/ar/LC_MESSAGES/djangojs.po:9178]: no placeholder in source string

Comment (burhandodhy) [reply conf/locale/ar/LC_MESSAGES/djangojs.po]: done

File: conf/locale/ar/LC_MESSAGES/djangojs.po (hunk 7 above)
Comment (burhandodhy) [reply conf/locale/ar/LC_MESSAGES/djangojs.po]: done

File: conf/locale/ar/LC_MESSAGES/djangojs.po (hunk 6 above)
Comment (burhandodhy) [reply conf/locale/ar/LC_MESSAGES/djangojs.po]: done

File: conf/locale/ar/LC_MESSAGES/djangojs.po (hunk 5 above)
Comment (burhandodhy) [reply conf/locale/ar/LC_MESSAGES/djangojs.po]: done

File: conf/locale/ar/LC_MESSAGES/djangojs.po (hunk 4 above)
Comment (burhandodhy) [reply conf/locale/ar/LC_MESSAGES/djangojs.po]: done

**Pull request merged**